analyticsdataserver/settings/base.py

```diff
@@ -39,21 +39,6 @@
 MANAGERS = ADMINS
 ########## END MANAGER CONFIGURATION
 
-
-########## DATABASE CONFIGURATION
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#databases
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.',
-        'NAME': '',
```

Comment:
Lets put CACHES and DATABASES in the base.py, and not change the local.py for now.